### PR TITLE
TRIVIAL: Update translations

### PR DESCRIPTION
**Pull request by petrjanu**
---

Supported PR commands:

| Command         | Description            |
| --------------- | ---------------------- |
| `ok to test`    | Re-run standard checks |
| `extended test` | BackstopJS tests       |

---

# PR Checklist

-   [ ] commit messages adhere to the [commit message guidelines](https://github.com/gooddata/gooddata-ui-sdk/blob/master/docs/contributing.md#what-should-the-commits-look-like)
-   [ ] `check` passes
-   [ ] `check-extended` passes
-   [ ] `rush change` [was run if applicable](https://github.com/gooddata/gooddata-ui-sdk/blob/master/docs/contributing.md#how-do-i-describe-my-changes-for-the-changelog)

### CODE REVIEW

File: libs/sdk-ui/src/base/localization/bundles/de-DE.json [hunk 1]
```diff
@@ -11,7 +11,7 @@
     "visualizations.totals.dropdown.title.min": "Min",
     "visualizations.totals.dropdown.title.max": "Max",
     "visualizations.totals.dropdown.title.nat": "Rollup (Gesamt)",
-    "visualizations.totals.dropdown.tooltip.nat.disabled.mvf": "Rollup (Gesamt) Aggregation ist nicht verfügbar, wenn die Betrachtung nach Messwert gefiltert wird. Um Rollup (Gesamt) zu verwenden, entfernen Sie alle Messwertfilter aus der Filterleiste.",
+    "visualizations.totals.dropdown.title.nat.disabled.tooltip": "Rollup (Gesamt) Aggregation ist nicht verfügbar, wenn die Betrachtung nach Messwert gefiltert wird. Um Rollup (Gesamt) zu verwenden, entfernen Sie alle Messwertfilter aus der Filterleiste.",
```
Comment (petrjanu) [inline libs/sdk-ui/src/base/localization/bundles/de-DE.json:14]: @xMort this rename of the key is intended?

**Review by xMort — APPROVED** [review]

**Comment by xMort** [comment]
LGTM

**Comment by gdgate** [comment]
Build in pipeline check aborted.

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-fossa-scanning-zuul-el7/359/ : SUCCESS in 22s (non-voting)
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-merge-compliance-check/615/ : SUCCESS in 12s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/2489/ : SUCCESS in 13m 56s

**Pull request merged**